packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -684,6 +688,74 @@ const Services = WebexPlugin.extend({
       .then(({body}) => this._formatReceivedHostmap(body));
   },
 
+  /**
+   * Initialize the discovery services and the whitelisted services.
+   *
+   * @returns {void}
+   */
+  initConfig() {
+    // Get the catalog and destructure the services config.
+    const catalog = this._getCatalog();
+    const {services} = this.webex.config;
+
+    // Validate that the services configuration exists.
+    if (services) {
+      // Check for discovery services.
+      if (services.discovery) {
+        // Format the discovery configuration into an injectable array.
+        const formattedDiscoveryServices = Object.keys(services.discovery)
+          .map((key) => ({
+            name: key,
+            defaultUrl: services.discovery[key]
+          }));
+
+        // Inject formatted services into services catalog.
+        catalog.updateServiceUrls('discovery', formattedDiscoveryServices);
+      }
+
+      // Check for allowed host domains.
+      if (services.allowedDomains) {
+        // Store the allowed domains as a property of the catalog.
+        catalog.setAllowedDomains(services.allowedDomains);
+      }
+
+      // Set `validateDomains` property to match configuration
+      this.validateDomains = services.validateDomains;
+    }
+  },
+
+  /**
+   * Make the initial requests to collect the root catalogs.
+   *
+   * @returns {Promise<void, Error>} - Errors if the token is unavailable.
+   */
+  initServiceCatalogs() {
+    this.logger.info('services: initializing initial service catalogs');
+
+    // Destructure the credentials plugin.
+    const {credentials} = this.webex;
+
+    // Init a promise chain to catch errors.
+    return Promise.resolve()
```

Comment:
this could be `return Promise.resolve(credentials.getOrgId())` so you don't need an empty resolve

Reply:
Good find, implementing this.